L1Trigger/L1THGCal/interface/concentrator/HGCalConcentratorSuperTriggerCellImpl.h

```diff
@@ -29,13 +29,15 @@ class HGCalConcentratorSuperTriggerCellImpl {
     oneBitFraction,
     equalShare,
   };
+  enum SubDetectorType { hgcal_silicon_CEE, hgcal_silicon_CEH, hgcal_scintillator };
```

Comment:
I'm wondering if this enum would not be better in the trigger tools header? Since it is currently defined in several places.